app/build.gradle

```diff
@@ -49,8 +49,8 @@ android {
 
     defaultConfig {
         applicationId "augsburg.se.alltagsguide"
-        versionCode 1
-        versionName "buddy"
+        versionCode 55564
```

Comment:
we still deploy via buddybuild, dont we? if so, this version code can be ignored as buddybuild will set it